app/views/viewall/students.html.erb

```diff
@@ -1,2 +1,40 @@
-<h1>Viewall#show</h1>
-<p>TODO: by hiten</p>
+<h2 align="center">Complete Student Member Status</h2>
+<hr>
+
+<table class="table table-justified">
+	<thead>
+		<td>id</td>
+		<td>name</td>
+		<td>total alumni alloted</td>
+		<td>response status</td>
+		<td>search status</td>
+	</thead>
+	<tbody>
+		<% @allStudents.each do |student| %>
+		<% @searchAdd = 0 %>
+		<% @responseAdd = 0 %>
+		<tr>
+			<% allties = TieAlumniWithStudentMember.where({ :StudentMember_id => "#{student.id}" }) %>
+				<% if allties.length >= 1 %>
+					<% allties.each do |tie| %>
+					<% stat = AlumniStatus.where({ :Alumni_id => "#{tie.Alumni_id}"}) %>
+						<% if stat[0].search == "success" %>
+						<% @searchAdd = @searchAdd + 1 %>
+						<% end %>
+						<% if stat[0].response == "success" %>
```

Comment:
Check the enum mentioned above and change `success` to something that is a member of the enum.